<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookRtnDialog</class>
 <widget class="QDialog" name="BookRtnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="idx_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入图书编号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>查 询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rtn_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>归 还</string>
   </property>
  </widget>
  <widget class="QPushButton" name="break_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>损 毁</string>
   </property>
  </widget>
  <widget class="QTableView" name="search_res_tbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="break_btn_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
